README.md

```diff
@@ -21,14 +23,48 @@ npm install @tbd54566975/dwn-sdk-js
 
 [API docs](https://tbd54566975.github.io/dwn-sdk-js/)
 
+Some projects that use this library: 
+
+* [Example CLI](https://github.com/TBD54566975/dwn-cli)
+* [Example with a web wallet](https://github.com/TBD54566975/incubating-web5-labs/)
+* [Server side aggregator](https://github.com/TBD54566975/dwn-server)
+
 ```javascript
-import { Dwn } from '@tbd54566975/dwn-sdk-js';
 
-// cool things
+import { Dwn, RecordsWrite, RecordsQuery } from '@tbd54566975/dwn-sdk-js';
+
+export const dwn = await Dwn.create({});
+
+....
+const query = await RecordsWrite.create({
+  data                        : randomBytes(randomInt(50, 500)),
+  dataFormat                  : 'application/json',
+  published                   : true,
+  protocol                    : 'yeeter',
+  schema                      : 'yeeter/post',
+  authorizationSignatureInput : signatureMaterial
+});
+
+let result = await dwn.processMessage(didState.did, query.toJSON());
```

Comment:
Method signature changed to take on data stream. Will pull down the branch and update the PR.